plugins/kubernetes/app/models/kubernetes/role.rb

```diff
@@ -40,7 +40,7 @@ class Role < ActiveRecord::Base
 
     # create initial roles for a project by reading kubernetes/*{.yml,.yaml,json} files into roles
     def self.seed!(project, git_ref)
-      kubernetes_config_files_in_repo(project, git_ref).each do |config_file|
+      kubernetes_config_files_in_repo(project, git_ref, ignore_errors: true).each do |config_file|
```

Comment:
while seeding we take what we can, users cannot deploy with missing roles anyway ...
but I guess we might as well show the error right there instead of letting you get into a weird state where you cannot deploy ...